src/style.css

```diff
@@ -1 +1,54 @@
 /* add styles here */
+
```

Comment:
The use of the universal selector `*` is against the guidelines as it can impact performance. Consider removing or replacing it with more specific selectors.